mon-pix/tests/acceptance/start-campaigns-test.js

```diff
@@ -82,8 +82,6 @@ describe('Acceptance | Campaigns | Start Campaigns', function() {
               // given
```

Comment:
Il ne semble pas y avoir de test qui couvre le nouveau comportement: l'identifiant reste alimenté après avoir tenté de s'inscrire avec adresse email existante
Est-ce dû au fait que ce n'est pas testable ?

Reply:
L'identifiant est un champ calculé et du coup est toujours affiché quand on switch, mais pas envoyé lors d'une inscription avec email. Voir json envoyé
`"email":"userPix1@example.net","username":null,"password":"pix123"`
Le fix a été drivé par les tests des scénarios nominaux pour checker la non régression. Il n'y a donc pas de changement de comportement mais plus un scénario tordu détecté par Lise qui est fixée. Un test BDD why not pour ce use case, mais clairement je pense qu'on va faire une passe sur tout le système d'authentification et d'inscription avec PIX account et donc cette partie aussi. Ce qui rend ce scénario non éligible à un test BDD car pas temporaire à mon sens

Reply:
Il n'y a pas de code temporaire…

Reply:
Dans start-campaigns-test.js il n'y a pas de tests vérifiant que les "inputs" du formulaire d'inscription ne sont pas perdus en cas d'erreurs 4xx.

Reply:
Vu avec la team accès, on rajoute un test de non régression qui valident les inputs avec une erreur 4XX, @jonathanperret si t'as une sugesstion de type de test adapté, n'hésites pas

Reply:
Un test d'acceptance qui reproduit le scénario du bug a été rajouté et qui valide que les inputs (login ,email,password) ne se reset plus.